src/utils/datetime_test.cljs

```diff
@@ -189,3 +192,75 @@
                                               "it"
                                               #'utils.datetime/medium-date-time-format)]
        (is (= (datetime/day-relative epoch) "01 gen 1970, 12:00:00 AM")))))
+
+(deftest format-mute-till-test
+  (let [remove-msecs              #(string/replace % #"\.\w*Z" "Z")
+        time-str-to-obj           #(t.format/parse (remove-msecs (time-coerce/to-string %)))
+        curr-time                 (t/now)
+        custom-HH-MM-formatter    (t.format/formatter "HH:mm")
+        custom-DD-formatter       (t.format/formatter "DD")
+        get-hh-mm                 #(t.format/unparse custom-HH-MM-formatter %)
+        get-day                   #(t.format/unparse custom-DD-formatter %)
+        get-week-day              #(->> %
+                                        t/day-of-week
+                                        (get constants/int->weekday))
+        mock-today                (t.format/unparse (t.format/formatters :date-time-no-ms) curr-time)
+        in-n-days                 #(-> (time-str-to-obj mock-today)
+                                       (t/plus (t/days %)))
+        in-n-minutes              #(-> (time-str-to-obj mock-today)
+                                       (t/plus (t/minutes %)))
+        in-n-hours                #(-> (time-str-to-obj mock-today)
+                                       (t/plus (t/hours %)))
+        mock-tomorrow             (in-n-days 1)
+        mock-in-two-days          (in-n-days 2)
+        mock-in-three-days        (in-n-days 3)
+        mock-in-four-days         (in-n-days 4)
+        mock-in-five-days         (in-n-days 5)
+        mock-in-six-days          (in-n-days 6)
+        mock-in-15-minutes        (in-n-minutes 15)
+        mock-in-1-hour            (in-n-hours 1)
+        mock-in-8-hour            (in-n-hours 8)
+        get-month-day-int         #(int (get-day %))
+        today?                    (fn [mocked curr-time]
+                                    (=
+                                     (t.format/unparse (t.format/formatter "MM:DD") mocked)
+                                     (t.format/unparse (t.format/formatter "MM:DD") curr-time)))
+        tomorrow?                 (fn [mocked curr-time]
+                                    (some #(= %
+                                              (-
+                                               (int (get-month-day-int mocked))
+                                               (int (get-month-day-int curr-time))))
+                                          [1 30 29 27]))
```

Comment:
31 - 1 is 30
30 - 1 is 29
28 - 1 is 27
7 - 1 is 1
To check if mocked date is tomorrow, According to calendar month